client/my-sites/hosting/phpmyadmin-card/index.js

```diff
@@ -0,0 +1,78 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import MaterialIcon from 'components/material-icon';
+import Button from 'components/button';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getHttpData, requestHttpData } from 'state/data-layer/http-data';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const requestId = siteId => `pma-link-request-${ siteId }`;
+
+export const requestPmaLink = siteId =>
+	requestHttpData(
+		requestId( siteId ),
+		http(
+			{
+				method: 'GET',
+				path: `/sites/${ siteId }/hosting/pma`,
+				apiNamespace: 'wpcom/v2',
+			},
+			{}
+		),
+		{
+			fromApi: () => ( { pmaUrl } ) => {
+				return [ [ requestId( siteId ), { pmaUrl } ] ];
+			},
```

Comment:
For me this only works the first time the button is clicked because the first api call is cached by default, so subsequent dispatches of the action don't result in a component rerender and useEffect being fired. Setting freshness property of RequestHttpDataOptions to 0 seems to fix the issue for me.  Probably no panic to fix here at this stage as we may not go with this approach for pma login, but handy to know if we use this pattern elsewhere.

```suggestion
			},
			freshness: 0,
```